reco_utils/tuning/nni/nni_utils.py

```diff
@@ -141,8 +142,9 @@ def start_nni(config_path, wait=WAITING_TIME, max_retries=MAX_RETRIES):
         max_retries (int): max number of retries
     """
     nni_env = os.environ.copy()
-    nni_env["PATH"] = sys.prefix + "/bin:" + nni_env["PATH"]
-    proc = subprocess.run(["nnictl", "create", "--config", config_path], env=nni_env)
+    nni_env['PATH'] = sys.prefix + '/bin:' + nni_env['PATH']
+    proc = subprocess.run([sys.prefix + '/bin/nnictl', 'create', '--config', config_path], env=nni_env)
```

Comment:
not the most desirable way to interact with nnictl. Would be nice if there was a python wrapper included with nnictl so that from here it could be called like a python package.

Reply:
yeah agree, afaik in nni they don't have a wrapper like that at the moment